### ISSUE TITLE
chore: data collection and processing

### ISSUE THREAD

**Issue by dannyl1u** [dannyl1u/doppelganger#23]

**Comment by dannyl1u** [comment]
assigned to @pigeonzow

**Comment by PigeonZow** [comment]
Initial scraping of issues
Didn't check how accurate the duplicate tagging yet

[open-webui_issues.csv](https://github.com/dannyl1u/doppelganger/files/15211755/open-webui_issues.csv)

**Comment by PigeonZow** [comment]
k I messed up and need to restrict LLM output LOL

**Comment by amiicao** [comment]
@PigeonZow  what's the status

**Comment by dannyl1u** [comment]
Assigned to @PigeonZow

**Comment by dannyl1u** [comment]
@PigeonZow if you could upload the script you had, would be greatly appreciated!

**Comment by PigeonZow** [comment]
hi danny! unfortunately I cannot proceed on this issue as there's an unmarked duplicate in #40 !

**Comment by amiicao** [comment]
@PigeonZow  it appears @PigeonZow  is blocked on #40 by @PigeonZow. Can you please explain?

**Comment by PigeonZow** [comment]
Hmmm yes very interesting - should I create a new issue to address this?
